cloud/src/autogluon/cloud/predictor/predictor.py

```diff
@@ -691,10 +736,11 @@ def predict(
             logger.warning('Providing a custom predictor_cls could break the deployment. Please refer to `AutoGluonBatchPredictor` for how to provide a custom predictor')
             predictor_cls = user_predictor_cls
 
-        split_type = kwargs.pop('split_type', None)
         content_type = kwargs.pop('content_type', None)
-        if not split_type:
```

Comment:
Can we just use `Line` as a default `split_type` vs passing `None` and then converting into default?

Reply:
Discussed offline. Will resolve